src/averageAges.js

```diff
@@ -15,11 +15,20 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let man = people.filter(person => person.sex === 'm');
+
+  century && (
```

Comment:
let's check if `century` is defined inside `filter`s callback to avoid an additional iterating
fix in other functions too